src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -506,6 +506,50 @@ function LazyArrays.cache_filldata!(P::SemiclassicalJacobiFamily, inds::Abstract
     P
 end
 
+###
+# here we construct hierarchies of c weight sums by means of contiguous recurrence relations
+###
+
+mutable struct SemiclassicalJacobiCWeightFamily{T, C} <: AbstractVector{SemiclassicalJacobiWeight{T}}
```

Comment:
This time is redundant, isn't it the same as `SemiclassicalJacobiFamily{T, T, T, C}`?

Reply:
I am confused, how is it the same? This is a vector containing weights with a range in the C index, the other is a vector of the polynomial bases themselves

Reply:
the goal is to compute `sum.(::Vector{SemiclassicalJacobiWeight})` using the contiguous recurrence in a single run when the entries only differ in the c parameter.